Resources/Prototypes/Entities/Clothing/Belt/belts.yml

```diff
@@ -478,6 +478,17 @@
   - type: Clothing
     sprite: Clothing/Belt/securitywebbing.rsi
 
+- type: entity
+  parent: ClothingBeltStorageBase
```

Comment:
this should probably parent off another webbing

Reply:
> this should probably parent off another webbing

so after all, webbings has such a postscript in the line parent

Reply:
> this should probably parent off another webbing

I checked and all webbings refer to ClothingBeltStorageBase in the parent line, do I really need to change this line?